curriculum/challenges/english/02-javascript-algorithms-and-data-structures/regular-expressions/restrict-possible-usernames.english.md

```diff
@@ -35,6 +35,10 @@ tests:
     testString: assert(!userCheck.test("007"), 'Your regex should not match <code>007</code>');
   - text: Your regex should not match <code>9</code>
     testString: assert(!userCheck.test("9"), 'Your regex should not match <code>9</code>');
+  - text: Your regex should not match <code>A1</code>
+    testString: 'assert(!userCheck.test("A1"), 'Your regex should not match <code>A1</code>');'
```

Comment:
remove `'` at the begin and end of the line.

Reply:
thanks, have removed them